### PR TITLE
Bib simulation

### PR DESCRIPTION
**Pull request by gianelle**
Configuration files for BIB simulation

### CODE REVIEW

File: simulation/001-BIB/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Configuration file used to simulate a bunch crossing starting from a seed (generated for example with MARS15)
+
+- Geometry: MuColl_v1/MuColl_v1.xml 
+- Physics.list: QGSP_BERT
+
+## Trimmed process
+
+The two bunch crossed (mu+ and mu-) are then overlaid to originated a single BIB. As signal it uses a single muon.
+
+During this process these time cuts are applyed **(-6σ ; +8σ)**:
```
Comment (gridge) [inline simulation/001-BIB/README.md]: we should probably specify that the units are `ns`

**Comment by gridge** [comment]
Now that I'm thinking about it, I wonder if with these MRs we should update the table directly in the main README.md file... it might be easier/faster to keep updated since we ensure that every MR updates that too (we could move that to another file if it becomes big..). what do you think?

**Comment by gridge** [comment]
I also think that the small reconstruction configuration would better be put in a reconstruction/001-BIB folder, since it is a separate step.. what do you think?

**Comment by gianelle** [comment]
> Now that I'm thinking about it, I wonder if with these MRs we should update the table directly in the main README.md file... it might be easier/faster to keep updated since we ensure that every MR updates that too (we could move that to another file if it becomes big..). what do you think?

The fact is that we should not have a MR for each new samples... If a sample uses an existing configuration, we do not need  a MR for it... or we have to create a MR for each new samples just with an update README.md.

Another point.. it is not simple to update a txt table! I prefer if we required, before to approve a MR, that an entry is added to the confluence table, or if the reviewer take care of it (if, for example, user has not the permission on confluence)

**Review by gridge — APPROVED** [review]

**Pull request merged**